styles/style.css

```diff
@@ -67,11 +97,16 @@ button {
 /* Jumbotron */
 
 .jumbotron {
-  background-image: url("../images/header.jpg");
+  background-image: url("/images/header-bike.jpg");
   border-radius: 0;
```

Comment:
The picture didn't show for me here, I think because the folder `images` is in a different directory to this file `style.css`, so it needs the `../` at the start

Reply:
done